packages/gatsby-transformer-raml-legacy/gatsby-node.js

```diff
@@ -11,6 +11,7 @@ const defineSchema = require('./src/schema/define-schema');
 
 async function onCreateNode(
   { node, actions, createNodeId, createContentDigest },
+  // Plugin options passed through `{ transformerRaml: {} }`
```

Comment:
The plugin should not care how the option is passed to it, so I think think comment can be confusing.

Reply:
Right, we can remove it then.